modules/ROOT/nav.adoc

```diff
@@ -11,19 +11,16 @@
   *** xref:troubleshoot-nfs.adoc[Troubleshoot NFS Errors]
  ** xref:prereq-gravity-check.adoc[Preinstallation Verification]
 * xref:install-workflow.adoc[Install Anypoint Platform PCE]
- ** xref:install-installer.adoc[Run the GUI Installer]
- ** xref:install-auto-install.adoc[Run the CLI Installer]
-* xref:config-workflow.adoc[Post-installation Tasks]
+* xref:upgrade.adoc[Upgrade Anypoint Platform PCE]
+* xref:install-uninstall-reinstall.adoc[Uninstall Anypoint Platform PCE]
+* xref:operating-about.adoc[Manage Anypoint Platform PCE]
```

Comment:
After moving the `Post-installation Tasks` here, the list of topics became too many.
I'd add a new section in the root called `Configure Anypoint Platform PCE` that contains:
- `Configure an External Identity Provider in Anypoint Platform PCE`
- `Configure SMTP in Anypoint Platform PCE`
- `Configure DNS on Anypoint Platform PCE`
- `Configure NFS Settings for Anypoint Private Cloud`
- `Configure Security for Anypoint Platform PCE`
- `Configure a Disclaimer for Anypoint Private Cloud`

Matching what's in:
![image](https://user-images.githubusercontent.com/473061/84414020-3fc57180-abc6-11ea-871d-1522a4ad3801.png)

The other topics can stay under `Manage Anypoint Platform PCE`.
And this page will need an update https://github.com/mulesoft/docs-private-cloud/blob/remove-install-upgrade-ph/modules/ROOT/pages/operating-about.adoc.

Reply:
Should we also move the "Delete Local User" page under the new "configure" root?

The new root page should have the same content as the "Close port 61009" page, except maybe remove the bullet about removing the port. I included that step in the internal installation instructions.

Reply:
Made some updates.

Reply:
Removed the uninstall info.